src/app/shared/conditional-show/conditional-show.directive.ts

```diff
@@ -100,6 +100,13 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
 
   private show() {
     this.el.nativeElement.hidden = false;
+    if (this.caseField && this.caseField.field_type && this.caseField.field_type.type !== 'Collection') {
+      if (this.el.nativeElement.closest('ccd-write-complex-type-field')) {
```

Comment:
this way of testing for a complex type is a bit fragile

Reply:
Done.